wfepy/workflow.py

```diff
@@ -125,13 +127,15 @@ def load(self, file_path):
 
     def dump(self, file_path):
```

Comment:
Do we actually use this to store the execution state?

I see only `state.json` in our Jira issues but it does not contain `context` only `state` (and some customized attributes).

Reply:
No. I think our codes override it. It contains state, subtask, data, etc. I am thinking to change our codes as well.